.github/workflows/wheels.yml

```diff
@@ -72,6 +72,8 @@ jobs:
           python: "pp38"
     env:
       IS_32_BIT: ${{ matrix.buildplat[1] == 'win32' }}
+      # Note: should sync this with the universal2 job below
+      MACOSX_DEPLOYMENT_TARGET: '10.9'
```

Comment:
Isn't this incorrect for `arm64`? It may work, but I think it'd be better to set it to `11.0` explicitly for `arm64`.